invenio_records_rest/config.py

```diff
@@ -363,3 +363,6 @@ def deleted_pid_error_handler(error):
     ),
 }
 """Handlers for ElasticSearch error codes."""
+
+RECORDS_REST_DEFAULT_SEARCH_RESULTS_SIZE = 10
```

Comment:
``RECORDS_REST_DEFAULT_SEARCH_RESULTS_SIZE`` -> 
``RECORDS_REST_DEFAULT_RESULTS_SIZE``? 

To make it a bit shorter?

Reply:
👍 done